src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        return "Certain machine started its work";
+    }
+
+    @Override
+    public void stopWork() {
+        return "certain machines stopped working";
```

Comment:
Similar to the 'doWork' method, 'stopWork' should print a message instead of returning a string. The message should be informative and indicate that the 'Truck' has stopped working.